Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -331,6 +331,32 @@ theorem bijective_comp (e : α ≃ β) (f : β → γ) : Bijective (f ∘ e) ↔
 theorem comp_bijective (f : α → β) (e : β ≃ γ) : Bijective (e ∘ f) ↔ Bijective f :=
   EquivLike.comp_bijective f e
 
+@[simp]
+theorem extend_apply {f : α ≃ β} (g : α → γ) (e' : β → γ) (b : β) :
+    extend f g e' b = g (f.symm b) := by
+  rw [← f.apply_symm_apply b, f.injective.extend_apply, apply_symm_apply]
+
+@[simp]
+theorem extend_id {α : Sort u} {γ : Sort w} (f : α → γ) (g : α → γ) :
+    extend id f g = f :=
+  funext <| injective_id.extend_apply f _
```

Comment:
This should be next to `Injective.extend_apply`, it doesn't need to be in this file which is about `Equiv`. Same for the lemma below.